server/portal/apps/workspace/api/views.py

```diff
@@ -247,21 +247,31 @@ def post(self, request, *args, **kwargs):
 
             if settings.DEBUG:
                 wh_base_url = settings.WH_BASE_URL + '/webhooks/'
-                # jobs_wh_url = settings.WH_BASE_URL + reverse('webhooks:jobs_wh_handler')
+                jobs_wh_url = settings.WH_BASE_URL + reverse('webhooks:jobs_wh_handler')
             else:
                 wh_base_url = request.build_absolute_uri('/webhooks/')
-                # jobs_wh_url = request.build_absolute_uri(reverse('webhooks:jobs_wh_handler'))
+                jobs_wh_url = request.build_absolute_uri(reverse('webhooks:jobs_wh_handler'))
 
             job_post['parameterSet']['envVariables'] = job_post['parameterSet'].get('envVariables', []) + [{'key': '_webhook_base_url', 'value':  wh_base_url}]
 
             portal_name = settings.PORTAL_NAMESPACE
             job_post['tags'] = job_post.get('tags', []) + [portal_name]
 
-            # TODOv3 Webhooks/notifications continues
-            # job_post['notifications'] = [
-            #     {'url': jobs_wh_url,
-            #      'event': e}
-            #     for e in settings.PORTAL_JOB_NOTIFICATION_STATES]
+            # TODOv3: change ttlMinutes to be dynamic based on max job runtime
+            job_post["subscriptions"] = [
+               {
+                    "description": "Email notification",
+                    "enabled": False,
+                    "eventCategoryFilter": "JOB_NEW_STATUS",
+                    "ttlMinutes": 10,
```

Comment:
Do we know what `ttlMinutes` corresponds to exactly? Is there a reason 10 is better than 0, for example?

Reply:
From my testing it looks like `ttlMinutes` corresponds to how long this subscription will be active and send notifications. For example if we set `ttlMinutes` to 10 but the job runs for 15 minutes, we won't receive any notifications past the 10 minute mark. This value likely needs to be dynamic based on the job. Perhaps the `maxMinutes` value from a job submission can be used here?

Reply:
it'll be `maxMinutes` plus whatever time it takes to set up the job, queue the job, and archive things. so we might be unable to calculate that number. I wonder if CIC can help us out with a number or some other way to say, "we want everything">

Reply:
Interesting. I wonder if there's a number to suggest "no limit" for the subscription as long as the job state is non-terminal

Reply:
@shayanaijaz Perhaps we can ask in the tapis channel?

Reply:
Interestingly enough, the subscription still works fine if I don't specify `ttlMinutes` at all so I wonder if that is the way to get the "no limit". I will conform this with the tapis team.

Reply:
Sounds like ttl of `0` should suit our needs